qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -224,6 +379,35 @@ def __init__(self, parent: Instrument, name: str, channel: str) -> None:
         self.add_parameter('fastsweep',
                            parameter_class=LuaSweepParameter)
 
+        self.add_parameter('npts',
```

Comment:
Since the `npts` parameter only affects the number of points of the `TimeTrace`, maybe it should have a name signifying that, e.g. `time_trace_npts`? The same goes for `dt` below.